en/docs/reference/connectors/salesforce-connectors/salesforcebulk-v2-connector-example.md

```diff
@@ -159,44 +133,41 @@ Lets add the operations to the resources in `Salesforce` API
       </salesforce_bulkapi_v2.uploadJobData>
       ```
 
-  5. Drag and drop `closeJob` operation from **Salesforce_bulkapi_v2_Connector** section.
-      1. Double-click the operation to view its properties section.
-      2. In the 'General' section of the properties, select the Salesforce connection configuration you created.
-      3. In the 'Job ID' text box, enter the expression `$ctx:jobId`.
+  5. Select the `closeJob` operation from **Salesforce_bulkapi_v2_Connector** section.
+      1. In the 'Salesforce Configuration' section of the properties, select the Salesforce connection configuration you created.
+      2. In the 'Job ID' text box, enter the expression `$ctx:jobId`.
 
-  6. Drag and drop 'Respond' mediator.
+  6. Select the 'Respond' mediator.
   <a href="{{base_path}}/assets/img/integrate/connectors/salesforcebulk-v2/create-job-upload-data.png"><img src="{{base_path}}/assets/img/integrate/connectors/salesforcebulk-v2/create-job-upload-data.png" width="800"/></a>
 
 #### - /createJobAndUploadFile
 
   Users can utilize this resource to send CSV content for upload via a CSV file. The API will utilize an `File Connector` to store the CSV content in a `csvContent` property. The 'UploadJobData' operation will then upload the `csvContent`. After uploading the content, the `CloseJob` operation will be used to change the job status to `UploadComplete`.
   
-  1. Drag and drop `createJob` operation from **Salesforce_bulkapi_v2_Connector** section.
-    1. Double-click the operation to view its properties section.
-    2. In the 'General' section of the properties, select the Salesforce connection configuration you created.
-    3. In the properties section, under `Basic`, select `INSERT` in the Operation dropdown.
-    4. Input `Account` in the `Object` text box
-    5. Select `COMMA` in the `Column Delimeter` dropbox
-    6. Select `LF` or `CRLF` in the `Line Ending` dropbox based on your operating system. IF Windows : `CRLF`, for Unix-based systems : `LF`
-
-  2. Drag and drop a property mediator. Using this mediator we will extract the jobId from the response and will use it in other operations in this sequence.
+  1. Select the `createJob` operation from **Salesforce_bulkapi_v2_Connector** section.
+    1. In the 'Salesforce Configuration' section of the properties, select the Salesforce connection configuration you created.
+    2. In the properties section, under `Basic`, select `INSERT` in the Operation dropdown.
+    3. Input `Account` in the `Object` text box
+    4. Select `COMMA` in the `Column Delimeter` dropbox
+    5. Select `LF` or `CRLF` in the `Line Ending` dropbox based on your operating system. IF Windows : `CRLF`, for Unix-based systems : `LF`
```

Comment:
can we use other numbering methods for this? (e.g: alphabets or roman numbers). Consider other places as well.

Reply:
When using a,b,c, its not rendered properly. But when using 1,2,3,4 it is rendered as a,b,c,d in the view